playbooks/roles/common/tasks/main.yml

```diff
@@ -1 +1,3 @@
 - include: wait-nodes.yml
+- include: workaround-RBAC.yml
+- { include: untaint_master.yml, when: nodes_count == 1 }
```

Comment:
You forgot to write "but it doesn't matter" so I moved it to another pull request